test/test_tracer.cpp

```diff
@@ -1191,6 +1191,31 @@ TEST_TRACER("span extraction") {
             nullopt,
             "0000000000000000",  // expected_datadog_w3c_parent_id,
         },
+
+        {
+            __LINE__,
+            "malformed traceparent 1/x",
```

Comment:
It seems like this test would fit better in a previous test case, where you test error cases and assert that the expected_error_tag_value is `malformed_traceparent`. However this input would result in `invalid_version`

Reply:
Good catch, thank you @zacharycmontoya . Addressed in 071b25b60735bfdbf83b4cb1a63cc669e4e7c087